stdlib/public/SwiftRemoteMirror/SwiftRemoteMirror.cpp

```diff
@@ -20,7 +21,7 @@ using namespace swift::reflection;
 using namespace swift::remote;
 
 using NativeReflectionContext
-  = ReflectionContext<External<RuntimeTarget<sizeof(uintptr_t)>>>;
+  = swift::reflection::ReflectionContext<External<RuntimeTarget<sizeof(uintptr_t)>>>;
```

Comment:
Duplicate of https://connect.microsoft.com/VisualStudio/feedback/details/3116636/msvc-reports-ambiguous-symbol-error-for-friend-class-declaration-in-an-anonymous-namespace